src/main/java/core/basesyntax/model/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class Main {
+    public static final int ARRAY_LENGTH = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Random random = new Random();
+        for (int i = 0; i < ARRAY_LENGTH; i++) {
+            if (i > 2) {
```

Comment:
If you use ARRAY_LENGTH in your for loop, use it to define that half on the array in `if` statement.